src/com/nutrons/steamworks/RobotBootstrapper.java

```diff
@@ -4,33 +4,52 @@
 import com.nutrons.framework.StreamManager;
 import com.nutrons.framework.controllers.Talon;
 import com.nutrons.framework.inputs.WpiXboxGamepad;
+import edu.wpi.first.wpilibj.ADXRS450_Gyro;
 
 public class RobotBootstrapper extends Robot {
+    private Talon intakeController;
```

Comment:
Can't change due to the whole "Talon following another Talon" thing, which requires both to be talons, not LSCs